### ISSUE TITLE
Adding power usage sensor

### ISSUE THREAD

**Issue by knudsvik** [fondberg/easee_hass#129]
It would be awesome to be able to read the power usage charging EV's given the new energy management features released in Home Assistant 2021.8.

**Comment by rasssta** [comment]
More info: https://developers.home-assistant.io/docs/core/entity/sensor/#long-term-statistics

**Comment by olalid** [comment]
In contrast to what the subject says, it is actually the energy sensor(s) that requires a change, the power sensor is not used for this purpose.
Until this is fixed, it is possible to feed a utility meter from the easee integration and use the utility meter in the energy card in HA.

**Comment by knudsvik** [comment]
As far as I can see, neither current power usage (W) or energy (kWh) is available today in the Easee integration. Both of these can be used for energy management with different effort. Ideally both should be available!

**Comment by olalid** [comment]
Yes both power and energy sensors are available.
They are not enabled by default, but they can be turned on in the configuration dialog of the integration. 
What is missing is the correct attributes that it needs to have to be considered an energy sensor capable of acting as input to as described in the link posted by rasssta above, not the sensor itself.

**Comment by knudsvik** [comment]
Ahh, look at that. I was not aware, thanks. Should I just close this issue or do we want to rephrase it to cover the correct setup of attributes?

**Comment by rasssta** [comment]
Just change the title :-)
